docs/source/package_reference/mixins.mdx

```diff
@@ -5,20 +5,27 @@
 The `huggingface_hub` library offers a range of mixins that can be used as a parent class for your
 objects, in order to provide simple uploading and downloading functions.
 
-### PyTorch
+### Generic
 
 [[autodoc]] ModelHubMixin
+    - all
+    - _save_pretrained
+    - _from_pretrained
+
+### PyTorch
+
+[[autodoc]] PyTorchModelHubMixin
 
 ### Keras
 
+[[autodoc]] KerasModelHubMixin
```

Comment:
I placed the mixing at the top because it felt weird to see functions on a doc page about mixins :)